Wireframe/css/style.css

```diff
@@ -0,0 +1,97 @@
+
+body {
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: 100px;
+  margin-right: 100px;
+  margin-bottom: 0;
+  background: #f0efe7;
+  padding: 0;
+  height: 100%;
+}
+
+header{
+  text-align: center;
+  margin-bottom: 5px;
+ 
+}
+h1,h2{
+  margin:5px;
+ 
+}
+article{
+  padding-left: 20px;
+}
+
+article:first-child {
+  margin-bottom: 10px;
+  border: 1px solid white;
+  border-radius: 15px;
+  height: auto;
+}
+
+#article-one-h2-margin{
+  margin-left: 80px;
+
+}
+#article-first-child-title{
+  display: flex;
+}
+
+#other-article-title{
```

Comment:
This is a container which has 2 articles in it, and not title. What would you change the name to?

Reply:
Resolved by changing to: other-articles-container